sessionize/lib/src/main/sqldelight/co/touchlab/droidcon/db/Session.sq

```diff
@@ -45,6 +47,9 @@ WHERE sessionSpeaker.userAccountId = ?;
 updateRsvp:
 UPDATE session SET rsvp = ? WHERE id = ?;
 
+updateFeedBack:
+UPDATE session SET feedbackRating = ?, feedbackComment = ? WHERE id = ?;
```

Comment:
Does sqldelight sanitize input automatically?

Reply:
I'm not sure, I don't see any mention of it in the readme or medium posts. In that case I would assume no.

Reply:
I think since sqldelight generates prepared statements, we don't need to worry about this